website/docs/d/cd_tekton_pipeline_trigger.html.markdown

```diff
@@ -56,9 +59,9 @@ After your data source is created, you can read values from the following attrib
 * `name` - (String) Trigger name.
   * Constraints: The maximum length is `253` characters. The minimum length is `1` character. The value must match regular expression `/^([a-zA-Z0-9]{1,2}|[a-zA-Z0-9][0-9a-zA-Z-_.: \/\\(\\)\\[\\]]{1,251}[a-zA-Z0-9])$/`.
 
-* `properties` - (List) Optional trigger properties used to override or supplement the pipeline properties when triggering a pipeline run.
+* `properties` - (List) Optional trigger properties are used to override or supplement the pipeline properties when triggering a pipeline run.
   * Constraints: The maximum length is `1024` items. The minimum length is `0` items.
-Nested schema for **properties**:
+Nested scheme for **properties**:
```

Comment:
Why we changed from schema to scheme?

Reply:
That's what the SDK generator generated for us

Reply:
Which version of generator were used may be we need to talk to generator team because in Hashicorp world we call it as Schema

Reply:
Checked and it was `v3.72.0` which is an older version. I can revert the change to `scheme` in this PR. Will look into generating everything with a more recent version of the generator for a future PR

Reply:
PR updated: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5531/commits/c34af27aa39a3858a8d4036d1eca66c4c0e5dd55